react-native-purchases-ui/src/index.tsx

```diff
@@ -1,31 +1,39 @@
-import {NativeModules, Platform, requireNativeComponent, UIManager} from "react-native";
-import {PAYWALL_RESULT} from "@revenuecat/purchases-typescript-internal";
+import { NativeModules, Platform, requireNativeComponent, UIManager } from "react-native";
+import { PAYWALL_RESULT } from "@revenuecat/purchases-typescript-internal";
 
-export {PAYWALL_RESULT} from "@revenuecat/purchases-typescript-internal";
+export { PAYWALL_RESULT } from "@revenuecat/purchases-typescript-internal";
```

Comment:
Do we still need this if it's inside of `RevenueCatUI`?

Reply:
I believe so, we are doing that in purchases.ts with the enums as well, and I think it's for compatiblity with commonjs. @tonidero do you know?

Reply:
In `purchases.ts` we kept it because the enums were defined and exported in that file, so to keep compatibility with users possibly importing directly from there, we had to keep exporting them there... I don't think it will be needed in this case though? Afaik, it should be ok to have them as part of the class... But not 100% sure.